src/modality-specific-files/motion.md

```diff
@@ -0,0 +1,196 @@
+When using quaternions to represent orientations, the axial components that corresponds to the three spatial axes MUST be specified as "quat_x", "quat_y", "quat_z", and the non-axial component as "quat_w".
+
+| **Keyword** | **Description**                                                                                                                                             |
+| ----------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| x           | Position along the X-axis, or rotation about the X-axis among the Euler angles that represent the orientation, or magnetic field strength along the X-axis. |
+| y           | Position along the Y-axis or rotation about the Y-axis among the Euler angles that represent the orientation, or magnetic field strength along the Y-axis.  |
+| z           | Position along the Z-axis or rotation about the Z-axis among the Euler angles that represent the orientation, or magnetic field strength along the Z-axis.  |
+| quat_x      | Quaternion component associated with the X-axis.                                                                                                            |
+| quat_y      | Quaternion component associated with the Y-axis.                                                                                                            |
+| quat_z      | Quaternion component associated with the Z-axis.                                                                                                            |
+| quat_w      | Non-axial quaternion component.                                                                                                                             |
+| n/a         | Channels that have no corresponding spatial axis.                                                                                                           |
+
+### Restricted keyword list for channel type
+
+Restricted keyword list for column `type` in alphabetic order.
+Note that upper-case is REQUIRED:
+
+| **Keyword** | **Description**                                                                                                                                                                                                              | **Modalities to be shared with**         |
+| ----------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------- |
+| ACCEL       | Accelerometer channel, one channel for each spatial axis. Column `component` for the axis MUST be added to the `*_channels.tsv` file (x, y, or z).                                                                           | [fNIRS](.\near-infrared-spectroscopy.md) |
```

Comment:
why does this link to fNIRS now? 

Also, please note my comment at the end of this post: https://github.com/bids-standard/bids-specification/issues/1436#issuecomment-1470040467

Reply:
I think this is also the reason for the CI failure:

> [33mWARNING  -  [0mDocumentation file 'modality-specific-files/motion.md' contains a link to 'modality-specific-files/.\near-infrared-spectroscopy.md' which is not found in the documentation files.